frontend/src/features/public-form/PublicFormProvider.tsx

```diff
@@ -582,86 +545,25 @@ export const PublicFormProvider = ({
                     timestamp,
                   })
                 },
-                onError: (error) => {
-                  // TODO(#5826): Remove when we have resolved the Network Error
-                  datadogLogs.logger.warn(
-                    `handleSubmitForm: submit with virus scan`,
-                    {
-                      meta: {
-                        ...logMeta,
-                        responseMode: 'storage',
-                        method: 'axios',
-                        error,
-                      },
-                    },
-                  )
-
-                  // defaults to the safest option of storage submission without virus scanning
-                  return submitStorageFormWithFetch(
-                    enableEncryptionBoundaryShift,
-                  )
-                },
               },
             )
-          }
-
-          return (
-            (
-              enableEncryptionBoundaryShift
-                ? submitStorageModeClearFormMutation
-                : submitStorageModeFormMutation
-            )
-              .mutateAsync(
-                {
-                  ...formData,
-                  ...formPaymentData,
-                  publicKey: form.publicKey,
-                },
+            .catch(async (error) => {
+              datadogLogs.logger.warn(
+                `handleSubmitForm: submit with virus scan`,
                 {
-                  onSuccess: ({
-                    submissionId,
-                    timestamp,
-                    // payment forms will have non-empty paymentData field
-                    paymentData,
-                  }) => {
-                    trackSubmitForm(form)
-
-                    if (paymentData) {
-                      navigate(getPaymentPageUrl(formId, paymentData.paymentId))
-                      storePaymentMemory(paymentData.paymentId)
-                      return
-                    }
-                    setSubmissionData({
-                      id: submissionId,
-                      timestamp,
-                    })
-                  },
-                },
-              )
-              // Using catch since we are using mutateAsync and react-hook-form will continue bubbling this up.
-              .catch(async (error) => {
-                // TODO(#5826): Remove when we have resolved the Network Error
-                datadogLogs.logger.warn(`handleSubmitForm: ${error.message}`, {
                   meta: {
                     ...logMeta,
                     responseMode: 'storage',
                     method: 'axios',
-                    error: {
-                      message: error.message,
-                      stack: error.stack,
-                    },
+                    error,
                   },
-                })
+                },
+              )
+              showErrorToast(error, form)
```

Comment:
```suggestion
                if (/Network Error/i.test(error.message)) {
                                  axiosDebugFlow()
                                  return submitStorageFormWithFetch(
                                    enableEncryptionBoundaryShift,
                                  )
                                }
              showErrorToast(error, form)
```
should we add this back in?

Reply:
thanks! added back in 
b2010ae